python/prepare_release.py

```diff
@@ -0,0 +1,265 @@
+import os
+import uuid
+from typing import Tuple
+
+from github import Github
+
+from .common_tool_methods import *
+from enum import Enum
+
+MULTI_EXTENSION_SQL = "src/test/regress/sql/multi_extension.sql"
+CITUS_CONTROL = "src/backend/distributed/citus.control"
+MULTI_EXTENSION_OUT = "src/test/regress/expected/multi_extension.out"
+CONFIG_PY = "src/test/regress/upgrade/config.py"
+DISTRIBUTED_DIR_PATH = "src/backend/distributed"
+CONFIGURE_IN = "configure.in"
+CONFIGURE = "configure"
+CITUS_CONTROL_SEARCH_PATTERN = r"^default_version*"
+
+MULT_EXT_MAJOR_SEARCH_PATTERN = r"^\s*\d{1,2}\.\d{1,2}devel$"
+MULT_EXT_PATCH_SEARCH_PATTERN = r"^\s*\d{1,2}\.\d{1,2}.\d{1,2}$"
```

Comment:
1) MULT -> MULTI
2) MULT_EXT_PATCH_SEARCH_PATTERN -> MULT_EXT_MINOR_PATCH_SEARCH_PATTERN, since that pattern seems to be used for both minor & patch releases

Reply:
Fixed by https://github.com/citusdata/tools/pull/91/commits/a2e8aef98e41c785588ddb62c2ed0f902bee180d